cmake/cpp_bindgen.cmake.in

```diff
@@ -65,18 +65,23 @@ function(bindgen_enable_fortran_library target_name)
     set(__CPP_BINDGEN_CMAKE_DIR @__CPP_BINDGEN_CMAKE_DIR@)
 
     if(CMAKE_Fortran_COMPILER_LOADED)
-        if(NOT TARGET fortran_bindings_handle)
-            add_library(fortran_bindings_handle ${__CPP_BINDGEN_SOURCE_DIR}/cpp_bindgen/array_descriptor.f90 ${__CPP_BINDGEN_SOURCE_DIR}/cpp_bindgen/handle.f90)
-            target_link_libraries(fortran_bindings_handle PUBLIC cpp_bindgen_handle)
-            target_include_directories(fortran_bindings_handle PUBLIC ${CMAKE_CURRENT_BINARY_DIR})
-            include(${__CPP_BINDGEN_CMAKE_DIR}/fortran_helpers.cmake)
-            bindgen_enable_fortran_preprocessing_on_target(fortran_bindings_handle)
+        if(NOT TARGET cpp_bindgen_fortran_handle)
+            add_library(cpp_bindgen_fortran_handle ${__CPP_BINDGEN_SOURCE_DIR}/cpp_bindgen/array_descriptor.f90 ${__CPP_BINDGEN_SOURCE_DIR}/cpp_bindgen/handle.f90)
+            set(CPP_BINDGEN_Fortran_MODULES_DIRECTORY ${CMAKE_BINARY_DIR}/cpp_bindgen_modules
+                CACHE INTERNAL "Directory for Fortran modules of cpp_bindgen")
```

Comment:
Why does it need to be a cached variable, and not a normal variable?

Reply:
I think it needs to be, for the same reason as the `CPP_BINDGEN_${target_name}_fortran_bindings_path`. It is used in `install_cpp_bindgen_targets()` and therefore needs to be at least available in the parent scope, but since we don't know where the `install_cpp_bindgen_targets()` will be called we need to make it available to the whole project. Do you agree? I'll add an explanation.

Reply:
Yes you're right.